samples/snippets/src/test/java/pubsublite/QuickStartIT.java

```diff
@@ -133,14 +133,34 @@ public void testQuickstart() throws Exception {
     // Publish.
     PublisherExample.publisherExample(
         CLOUD_REGION, ZONE_ID, PROJECT_NUMBER, TOPIC_NAME, MESSAGE_COUNT);
-    assertThat(bout.toString()).contains("Published " + MESSAGE_COUNT);
+    assertThat(bout.toString()).contains("Published " + MESSAGE_COUNT + " messages.");
+
+    bout.reset();
```

Comment:
nit. its unclear that this means 'binary output'. consider doing a find-replace to stdout? the fact that its a binary stream seems to not be important to me, I could be wrong though.

Reply:
This works as intended actually.